src/main/java/io/neonbee/data/internal/metrics/DummyDataVerticleMetrics.java

```diff
@@ -0,0 +1,34 @@
+package io.neonbee.data.internal.metrics;
+
+import java.util.List;
+
+import io.micrometer.core.instrument.Tag;
+import io.vertx.core.Future;
+
+/**
+ * This implementation of the {@link DataVerticleMetrics} interface is used when metrics are disabled.
+ */
+public class DummyDataVerticleMetrics implements DataVerticleMetrics {
```

Comment:
What do you think about renaming this to NoopDataVerticleMetrics as this class is actually doing null operations. Dummy sounds more like it's doing something "simple", which is not true.

```suggestion
public class NoopDataVerticleMetrics implements DataVerticleMetrics {
```

Reply:
renamed